File: app/src/main/java/com/example/syl/pokedex/global/model/Pokemon.java
package com.example.syl.pokedex.global.model;

public class Pokemon {

    String name;
    Sprites sprites;

    Pokemon(String name, Sprites sprites) {
        this.name = name;
        this.sprites = sprites;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public Sprites getSprites() {
        return sprites;
    }

    public void setSprites(Sprites sprites) {
        this.sprites = sprites;
    }

    @Override
    public String toString() {
        return "Pokemon{" +
                "name='" + name + '\'' +
                ", sprites='" + sprites.toString() + '\'' +
                '}';
    }
}

File: app/src/main/java/com/example/syl/pokedex/global/model/Pokemon.kt
package com.example.syl.pokedex.global.model

class Pokemon internal constructor(var name: String, var sprites: Sprites) {

    override fun toString(): String {
        return "Pokemon{" +
                "name='" + name + '\''.toString() +
                ", sprites='" + sprites.toString() + '\''.toString() +
                '}'.toString()
    }
}
